### ISSUE TITLE
react-native-web not extended

### ISSUE THREAD

**Issue by jtag05** [Chion82/native-base-web#2]
First of all, I'd like to say that this library is wonderful and is definitely something I've been looking for. The only thing preventing me from using it is your usage of `react-native-web-extended` as a replacement for `react-native-web`. In my experience using a fork as a means of replacement can get messy as things are changed in the upstream repo. 

I suppose my question/proposed resolution would be: Is it possible for you to isolate the components that you've added to `react-native-web` in `react-native-web-extened` and package them as a separate dependency for this library, or offer your changes upstream as a pull request to the `react-native-web` repo?

Thanks again for all of the wonderful work. If you'd like any help with any of this, don't hesitate to ask.

**Comment by Chion82** [comment]
Agree. I've done the major migration to make `native-base-web` run on `react-native-web`. Read [Manual Setup](https://github.com/Chion82/native-base-web/blob/dev/README.md#manual-setup) for details.
As for `react-native-web-extended`, it has been merged with the latest upstream branch currently. 

However it is not appropriate to separate those extended components into another package, because it will be messy for isomorphic apps: 
Say, `import { TabBarIOS } from 'extended-components-package'` doesn't work for `react-native` projects.
By joining the extended components and APIs into the existing `react-native-web(-extended)`, `import { TabBarIOS } from 'react-native'` works on both native and web platforms and all we need is just a webpack resolving alias.

I think it's too early to create a pull request to offer those extended changes by now, because they are so experimental and may be changed a lot recently. Meanwhile you can feel free using `react-native-web`.

Thanks for your advices!
